docs/it-manual/sre-playbook/production-database-access.md

```diff
@@ -0,0 +1,23 @@
+# Production database access
+
+The CiviForm deployment system provides a mechanism for temporary and secure direct access to the production database via the [pgadmin tool](https://www.pgadmin.org/). This can be useful when investigating and/or remediating production incidents.
+
+**When accessing the database in this way, it is strongly recommended to do so on a trusted device and network
+or a cloud CLI tool such as [AWS CloudShell](https://aws.amazon.com/cloudshell/).
+Furthemore, do not interrupt the script while it is running (ctrl-c) either while it is running as this may prevent it from revoking the temporary security settings allowing connections from the public internet.**
```

Comment:
Yep this would be a good improvement, though I'm unlikely to get to it anytime soon. The code starts here before delegating to the cloud-specific version: https://github.com/civiform/cloud-deploy-infra/blob/main/cloud/shared/bin/pgadmin.py#L6